.tekton/client-server-cg-push.yaml

```diff
@@ -376,23 +356,21 @@ spec:
         operator: in
         values:
         - "false"
+    - name: run-unit-test
```

Comment:
We dont need to run the unit tests in these pipelines we only need to run them in `.tekton/cosign-pull-request.yaml` and `.tekton/cosign-push.yaml`

Reply:
Oh sry,  misunderstood that part :) I will change it :+1: